app/assets/stylesheets/base.scss

```diff
@@ -16,9 +16,15 @@ body {
   margin-left: 200px;
   padding-left: 20px;
   padding-right: 20px;
+}
 
-  &.collapsed-nav {
-    margin-left: 75px;
+/*
+  patternfly-sass hides the vertical-nav on smaller screens, see: https://github.com/patternfly/patternfly-sass/blob/5e7cce3445d5b1af1e16500695b9f33edc6a4f6c/assets/javascripts/patternfly-functions.js#L260
+  the react-container needs to be shifted when it happens.
+*/
+@media only screen and (max-width: 768px) {
```

Comment:
Can we put it in webpack and import the pf3 variable for the max-width?

Reply:
thanks, added those in my last commit

Reply:
Resolved